assets/sass/authorize-application/_googlesitekit-authorize-application.scss

```diff
@@ -16,10 +16,138 @@
  * limitations under the License.
  */
 
-.auth-app-card #approve {
-	background-color: $c-deep-blue;
+.authorize-application-php {
+	background-color: $c-authorize-application-background;
+
+	.wrap {
+		margin: 30px 0 0;
+
+		h1 {
+			// This is the Authorize Application screen title.
+			font-size: $fs-authorize-application-headline-sm;
+			line-height: $lh-authorize-application-headline-sm;
+			padding: 0;
+		}
+	}
+
+	.auth-app-card {
+		border: none;
+		margin: 37px 0 0;
+		padding: 0;
+	}
+
+	.title {
+		font-size: $fs-authorize-application-title-lg;
+		font-weight: $fw-medium;
+		line-height: $lh-authorize-application-title-lg;
+		margin: 19px 0;
+	}
+
+	p {
+		font-size: $fs-authorize-application-label-lg;
+		font-weight: $fw-medium;
+		line-height: $lh-authorize-application-label-lg;
+		margin: 19px 0 20px;
+	}
+
+	strong {
+		font-weight: $fw-medium;
+	}
+
+	.form-field {
+		margin: 20px 0 40px;
+		max-width: 512px;
+
+		label {
+			font-size: $fs-authorize-application-label-md;
+			font-weight: $fw-medium;
+			line-height: $lh-authorize-application-label-md;
+		}
+
+		input {
+			border: 1px solid $c-authorize-application-sys-light-outline;
+			border-radius: $br-xs;
+			font-size: $fs-authorize-application-body-md;
+			line-height: $lh-authorize-application-body-md;
+			margin: 8px 0 0;
+			padding: 12px 16px;
+			width: 100%;
+		}
+	}
+
+	.description {
+		font-size: $fs-authorize-application-label-md;
+		line-height: $lh-authorize-application-label-md;
+
+		strong {
+			display: block;
+			margin: 9px 0 0;
+			max-width: 512px;
+		}
+
+		code {
```

Comment:
We should set the background color to `#ededed` as per the Figme design.
![Screenshot 2024-04-25 at 5 52 07 PM](https://github.com/google/site-kit-wp/assets/24408261/45d557b8-d48b-4fec-af43-405777f01124)

Reply:
Actually, this already is `#ededed` if you inspect the resulting colour:

https://github.com/google/site-kit-wp/assets/18395600/562ab7e0-1430-4bac-a1c5-57b9802f9fb1

Reply:
Hmm, it's strange. I get `#efefef`.

https://github.com/google/site-kit-wp/assets/24408261/a432173b-d589-4bcd-b813-507feeb546d3

Reply:
Oh right, how odd! Well, best fix it then - I've pushed the required change. Thanks for the spot!